server/src/DTOs/DTOs.ts

```diff
@@ -0,0 +1,16 @@
+export type CreateMemberDTO = {
+  name: string
+  is_active: boolean
```

Comment:
I don't if we need this info for the front. The request will just return the active members

Reply:
I don't think I understand what u mean. This DTO is only for type-restriction purposes when creating a new Member. It does not interfere with the request for all members from the client (?)